addon/services/construction-project.js

```diff
@@ -166,6 +166,16 @@ export default class ConstructionProjectService extends GwrService {
     return projects;
   }
 
+  async removeProjectLink(projectId) {
+    const link = this.store
+      .peekAll("gwr-link")
+      .find(({ eproid }) => Number(eproid) === projectId);
+    await link.destroyRecord();
+    this.all.lastSuccessful.value = this.all.lastSuccessful.value.filter(
```

Comment:
NP: Similarly `this.all` is vague. `*allProjects` would be more clear.

Reply:
Leaving it as is since this convention is used across all gwr services (for the better or worse :shrug: )